lgn-provers/src/provers/v1/groth16/euclid_prover.rs

```diff
@@ -1,17 +1,17 @@
 use std::collections::HashMap;
 
 use anyhow::Result;
-use groth16_framework::Groth16Prover as InnerProver;
+use groth16_framework::Groth16Prover;
 use tracing::debug;
 
 use crate::params;
 
 #[derive(Debug)]
-pub struct Groth16Prover {
-    inner: InnerProver,
+pub struct Groth16EuclidProver {
+    inner: Groth16Prover,
 }
```

Comment:
These are unrelated changes, I just named all the provers `<type>(Dummy|Euclid)Prover` and made the log messages consistent.